documentation/meeting-logs/2023-11-13.md

```diff
@@ -0,0 +1,72 @@
+# Standup Meeting 9
+
+2023-11-13
+Sprint 3
+
+Meeting duration: 1 hour
+
+## Attendees
+
+All group members
+
+## Updates
+
+Dan:
+
+- Wrote all 10-integration test pr up and merged on the discussion
+  - [Integration Test PR](https://github.com/BarbzCodez/Spendr/pull/136)
+  - Ready to be merged
+  - [Integration Testing #109](https://github.com/BarbzCodez/Spendr/issues/109)
+- [Fix analytics endpoints](https://github.com/BarbzCodez/Spendr/pull/135)
+- Added the receipt image parsing
+  - [Receipt Parsing PR](https://github.com/BarbzCodez/Spendr/pull/130)
```

Comment:
Thanks for adding this- I forgot to mention it in the meeting.

Reply:
I just went through the prs merged and was like oh damn, should add this lol